dll/cpl/sysdm/lang/pt-PT.rc

```diff
@@ -242,23 +245,23 @@ BEGIN
     LTEXT "%s", IDC_DRIVE, 88, 80, 100, 9
     LTEXT "Espaço disponível:", IDC_STATIC, 20, 92, 60, 9
     LTEXT "%s", IDC_SPACEAVAIL, 88, 92, 104, 9
-    LTEXT "Tamanho inicial (MB):", -1, 22, 118, 75, 9
-    LTEXT "Tamanho máximo (MB):", -1, 22, 131, 75, 9
+    LTEXT "Tamanho &inicial (MB):", -1, 22, 118, 75, 9
```

Comment:
MS really uses &i here? They do typically recommend to avoid that. Have a pic?

Reply:
![VirtualBox_WinSrv2003_Memoria](https://github.com/reactos/reactos/assets/49964946/ce0d329b-c6f7-4a3d-a070-361b2dd64787)

Yes, on Win2003 Server Enterprise. But in WinXP there is no such accelerator.
Do you want me to remove these 3 accelerators?

Reply:
No don't remove the accels. But use "&Tamnho inicial" instead of Tamanho &initial" here. That avoids the ugly &i.